### ISSUE TITLE
Chunk options aren't preserved for SQL exercise chunks

### ISSUE THREAD

**Issue by gadenbuie** [rstudio/learnr#667]
Reported on RStudio Community at https://community.rstudio.com/t/sql-exercises-in-learnr-return-only-errors-still/130065 by @rmccreath

> I'm creating a {learnr} tutorial and looking to include SQL exercises. However, I'm getting a " The 'connection' option (DBI connection) is required for sql chunks." if I pass the connection variable in the RMarkdown with quotes. If I try to pass the connection variable without quotes, the app crashes with an unexpected symbol error.
> 
> I've attempted the test example given by [@grrrck](https://community.rstudio.com/u/grrrck) in the linked post and I'm using the latest GitHub version. Is there something else I'm missing or could give a try to get it working?
> 
> Thanks! 🙂 
> 

It appears that the chunk options for the original `sql` exercise chunk aren't preserved when we render the user's code. Instead I see

````
```{sql "db"}
SELECT * FROM films
```
````

where I expect to see (reprex below)

````
```{sql "db", connection="db_con"}
SELECT * FROM films
```
````

<details>
<summary>Reprex</summary>

````markdown
---
title: "SQL Demo"
output: learnr::tutorial
runtime: shiny_prerendered
---

```{r setup, include=FALSE}
library(learnr)

# install.packages("starwarsdb")
library(starwarsdb)
db_con <- starwars_connect()
```


## Topic 1

### Exercise

```{sql db, exercise = TRUE, connection = "db_con"}

```
````

</details>

**Comment by gadenbuie** [comment]
@rmccreath I have a proposed fix in #668 for the issue you reported. If you'd like to try it out, you can install that version of learnr and give it a spin. If you do, please let me know how it goes! 😄 

```r
remotes::install_github("rstudio/learnr@fix-chunk-knitr-options")
```

**Comment by rmccreath** [comment]
@gadenbuie tested on my original project and a couple of other test cases... and everything is working! 🎉 Thanks for the fix and for keeping me in the loop.
